docs/src/conf.py

```diff
@@ -82,6 +106,35 @@
 # src base directory
 base_dir = Path(__file__).absolute().parent
 
+# -- autoapi extension --------------------------------------------------------
+# See https://sphinx-autoapi.readthedocs.io/en/latest/reference/config.html
+#     https://github.com/readthedocs/sphinx-autoapi
+#
+source_code_root = (Path(__file__).parents[2]).absolute()
+module_dir = source_code_root / "src"
```

Comment:
@tkknight Could you move the `root_dir` and `module_dir` to live with the `base_dir`, thanks 👍